data/json/npcs/TALK_COMMON_MISSION.json

```diff
@@ -237,15 +245,13 @@
       {
         "text": "Maybe you can teach me something as payment?",
         "topic": "TALK_TRAIN",
-        "condition": { "or": [ "npc_train_skills", "npc_train_styles" ] },
-        "effect": "mission_reward"
+        "condition": { "or": [ "npc_train_skills", "npc_train_styles" ] }
```

Comment:
```
+        "effect": "clear_mission"
```

Reply:
Everything that leads into TALK_MISSION_SUCCESS has an effect:mission_success on it already. This clears the mission as a side effect, I think. I am pretty sure I tested the training reward option and the mission was cleared.